promql/bench_test.go

```diff
@@ -186,6 +186,10 @@ func rangeQueryCases() []benchCase {
 			expr:  "count({__name__!=\"\",l=\"\"})",
 			steps: 1,
 		},
+		// timestamp() function
```

Comment:
comment doesn't add much

Reply:
I agree - I was following the existing pattern in this test of including a comment before each group of test cases. Happy to remove it if you'd prefer.

Reply:
Under that pattern it could be something like "Functions which have special handling inside 'eval'".

Reply:
Makes sense, changed in d396282